odxtools/companydata.py

```diff
@@ -179,15 +187,10 @@ def from_et(et_element, doc_frags: List[OdxDocFragment]) -> "CompanyData":
 
             roles = rlist
 
-        team_members = et_element.find("TEAM-MEMBERS")
-        if team_members is not None:
-            tml = NamedItemList(short_name_as_id)  # type: ignore
-
-            for tm in team_members.iterfind("TEAM-MEMBER"):
-                tml.append(TeamMember.from_et(tm, doc_frags))
-
-            team_members = tml
-
+        team_members = [
+            TeamMember.from_et(tm, doc_frags)
+            for tm in et_element.iterfind("TEAM-MEMBERS/TEAM-MEMBER")
+        ]
```

Comment:
this is also an unrelated cleanup, but IMO it is trivial enough not to warrant a separate PR...